br/pkg/task/rawkv_config.go

```diff
@@ -112,6 +112,9 @@ func (cfg *RawKvConfig) parseDstAPIVersion(flags *pflag.FlagSet) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+	if len(originalValue) == 0 {
```

Comment:
`dst-api-version` design to default as the same to api version of TiKV.

Reply:
ok, the behaviour is changed. Keep empty valid and adjust the value after the cur api version got.